cli/demo_direct_call.sh

```diff
@@ -84,19 +84,21 @@ echo "* Send ${AMOUNTTRANSFER} funds from Alice's incognito account to Bob's inc
 $CLIENT trusted transfer ${ICGACCOUNTALICE} ${ICGACCOUNTBOB} ${AMOUNTTRANSFER} --mrenclave ${MRENCLAVE} --direct
 echo ""
 
+echo "* Waiting 3 seconds"
+sleep 3
+echo ""
+
```

Comment:
We wait until we execute the getters to ensure the sidechain block got imported and the balance updated.